cfme/cloud/stack.py

```diff
@@ -49,11 +48,14 @@ def edit_tags(self, tag, value):
                               'select_value': value},
              action=form_buttons.save)
         flash.assert_success_message('Tag edits were successfully saved')
+        self.get_tags(tag, value)
+
+    def get_tags(self, tag, value):
         sel.force_navigate('clouds_stack', context={'stack': self})
         row = sel.elements("//*[(self::th or self::td) and normalize-space(.)={}]/../.."
                      "//td[img[contains(@src, 'smarttag')]]".format(quoteattr("My Company Tags")))
-        tag = sel.text(row).strip()
-        assert tag == "Cost Center: Cost Center 001"
+        company_tag = sel.text(row).strip()
+        assert company_tag == tag.replace(" *", "") + ": " + value
```

Comment:
So to clarify....... ```get_tags``` should only be "getting the tag value/s"
An assertion to ensure the tag was changed to the right value, needs to be in the edit method.